packages/web/lib/hooks/index.ts

```diff
@@ -20,6 +20,7 @@ export const useUser = ({ redirectTo, redirectIfFound }: UseUserOpts = {}): {
 
   const [{ data, error, fetching }] = useGetMeQuery({
     pause: !authToken,
+    variables: { forLoginDisplay: true },
```

Comment:
Passing this here will cause it to be true every time someone calls `useUser`. This breaks the profile editing process. `forLoginDisplay` should only be `true` in `LoginButton`.

```suggestion
```